guides/micronaut-cloud-database-aws/micronaut-cloud-database-aws.adoc

```diff
@@ -0,0 +1,177 @@
+
+[source,bash]
+----
+# VPC, internet gateway and route table
+export VPC_ID=$(aws ec2 create-vpc --cidr-block 10.0.0.0/16 | jq -r '.Vpc.VpcId')
+export IG_ID=$(aws ec2 create-internet-gateway | jq -r '.InternetGateway.InternetGatewayId')
+aws ec2 attach-internet-gateway --internet-gateway-id $IG_ID --vpc-id $VPC_ID
+aws ec2 modify-vpc-attribute --enable-dns-hostnames --vpc-id $VPC_ID
+export RT_ID=$(aws ec2 describe-route-tables --filters "Name=vpc-id,Values=$VPC_ID" --query "RouteTables[].RouteTableId" --output text)
+aws ec2 create-route --route-table-id $RT_ID --destination-cidr-block 0.0.0.0/0 --gateway-id $IG_ID
+
+# Security group
+aws ec2 create-security-group --group-name micronaut-guides-mysql-sg --description "Security Group for the Micronaut MySQL guide" --vpc-id $VPC_ID
+export SG_ID=$(aws ec2 describe-security-groups --query 'SecurityGroups[?GroupName==`micronaut-guides-mysql-sg`].GroupId' --output text)
+aws ec2 authorize-security-group-ingress --group-id $SG_ID --protocol tcp --port 3306 --cidr $(curl ifconfig.me)/32
+
+# Subnets and subnet group
+export AZ_0=$(aws ec2 describe-availability-zones --filters "Name=state,Values=available" --query "AvailabilityZones[0].ZoneName" --output text)
+export AZ_1=$(aws ec2 describe-availability-zones --filters "Name=state,Values=available" --query "AvailabilityZones[1].ZoneName" --output text)
+export SN0_ID=$(aws ec2 create-subnet --vpc-id $VPC_ID --cidr-block 10.0.0.0/20 --availability-zone $AZ_0 | jq -r '.Subnet.SubnetId')
+export SN1_ID=$(aws ec2 create-subnet --vpc-id $VPC_ID --cidr-block 10.0.16.0/20 --availability-zone $AZ_1 | jq -r '.Subnet.SubnetId')
+aws ec2 modify-subnet-attribute --subnet-id $SN0_ID --map-public-ip-on-launch
+aws ec2 modify-subnet-attribute --subnet-id $SN1_ID --map-public-ip-on-launch
+aws rds create-db-subnet-group --db-subnet-group-name micronaut-guides-mysql-sng --db-subnet-group-description "DB subnet group for the Micronaut MySQL guide" --subnet-ids "$SN0_ID" "$SN1_ID"
+----
+
+=== Create the MySQL instance
+
+[source,bash]
+----
+aws rds create-db-instance \
+    --db-instance-identifier micronaut-guides-mysql \
+    --db-instance-class db.t2.micro \
+    --engine mysql \
+    --master-username admin \
+    --master-user-password secret99 \
+    --allocated-storage 20 \
+    --db-subnet-group-name micronaut-guides-mysql-sng \
+    --vpc-security-group-ids $SG_ID \
+    --publicly-accessible
+----
+
+You then need to wait for the instance to be available:
+
+[source,bash]
+----
+aws rds wait db-instance-available --db-instance-identifier micronaut-guides-mysql
+----
+
+Once available, you can configure environment variables for its hostname and port:
+
+[source,bash]
+----
+export MYSQL_HOST=$(aws rds describe-db-instances --query 'DBInstances[?DBInstanceIdentifier==`micronaut-guides-mysql`].Endpoint.Address' --output text)
+----
+
+Finally, test connectivity to the database using the MySQL client CLI:
+
+[source,bash]
+----
+mysql -u admin -psecret99
```

Comment:
instead of specifying the password from the command (which leaves the password in `~/.bash_history` or equivalent) have them run `mysql -u admin -p` and enter the password when prompted

Reply:
Makes sense. I wanted to deliver as much copy/paste-able content as possible, but this instance should be different